src/main/java/com/ays/backend/user/model/Type.java

```diff
@@ -0,0 +1,20 @@
+package com.ays.backend.user.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "type")
```

Comment:
Please be more consistent about the table names, the others are having suffix "s", this doesn't.